readme.md

```diff
@@ -91,7 +91,9 @@ spec :: forall input m. Monad m => F.Spec' DogForm Dog input m
 spec = F.defaultSpec { render = render, handleEvent = F.raiseResult }
   where
   render st@{ form } =
-    HH.form_
+    HH.form
+      [ HE.onSubmit F.submitPreventDefault
+      ]
```

Comment:
Use `submitPreventDefault` in the readme example to avoid confusion

Reply:
I can do this in a follow up PR if you'd like, but we really should switch all forms to be written this way

Reply:
I was thinking to do that follow up PR, if that's not a problem